docs/user/02-logs.md

```diff
@@ -170,45 +174,93 @@ Telemetry Manager supports different types of [Fluent Bit filter](https://docs.f
 
 ### Step 4: Add authentication details from Secrets
 
-Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets.
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets. At the moment, mTLS and Basic Authentication are supported.
 
-Using the **http** output definition and the **valueFrom** attribute, you can map Secret keys as in the following **http** output example:
+Using the **http** output definition and the **valueFrom** attribute, you can map Secret keys as in the following examples:
 
-```yaml
-kind: LogPipeline
-apiVersion: telemetry.kyma-project.io/v1alpha1
-metadata:
-  name: http-backend
-spec:
-  output:
-     http:
+<div tabs>
```

Comment:
Excellent idea to make the code samples expandable, much better reading experience!